rust/cloud-storage/rpc-interface/auth_service/justfile

```diff
@@ -0,0 +1,7 @@
+out_dir := "../../../../js/app/packages/codegen/" + file_name(justfile_directory())
```

Comment:
This won't build properly in CI since this is never called.

Reply:
This doesn't need to built in this PR, its not used until the next PR